module-system/sov-modules-macros/src/cli_parser.rs

```diff
@@ -1,7 +1,7 @@
 use quote::{format_ident, quote};
-use syn::{Data, DataEnum, DeriveInput, Fields, Ident, Type};
+use syn::{Data, DataEnum, DeriveInput, Fields, Ident};
 
-use crate::common::{extract_ident, StructFieldExtractor};
+use crate::common::StructFieldExtractor;
 
 pub(crate) struct CliParserMacro {
```

Comment:
Do we have a unit test for this macro?

Reply:
Updated the old test to work with the new structure!